pkgs/racket-doc/scribblings/reference/match.scrbl

```diff
@@ -256,8 +358,12 @@ In more detail, patterns match as follows:
          [(hash-table ("b" b) ("a" a)) (list b a)])
        ]}
 
- @item{@racket[(#,(racketidfont "hash-table") (_pat _pat) ...+ _ooo)]
-       --- Generalizes @racketidfont{hash-table} to support a final
+ @item{@racket[(#,(racketidfont "hash-table") (_pat _pat) ...+ _ooo)] ---
+       @bold{This pattern is deprecated because it has a bug.}
```

Comment:
I would prefer "because it can be incorrect".

Reply:
Changed!